services/provider/server/server.go

```diff
@@ -40,9 +40,11 @@ import (
 )
 
 const (
-	TicketAnnotation          = "ocs.openshift.io/provider-onboarding-ticket"
-	ProviderCertsMountPoint   = "/mnt/cert"
-	onboardingTicketKeySecret = "onboarding-ticket-key"
+	TicketAnnotation           = "ocs.openshift.io/provider-onboarding-ticket"
+	ProviderCertsMountPoint    = "/mnt/cert"
+	onboardingTicketKeySecret  = "onboarding-ticket-key"
+	consumerUUIDLabel          = "ocs.openshift.io/storageconsumer-uuid"
+	storageClassClaimNameLabel = "ocs.openshift.io/storageclassclaim-name"
```

Comment:
Chosen labels over annotation because the consumerID and claimName are added as the identification of the resources and if we add these informations as annotation we cannot do get or list operations based on the annotation.

Reply:
Ok